### DISCUSSION TITLE
Custimize resize stepper size to make resizing smoother.

### DISCUSSION THREAD

**Discussion by DimitarDevP** [pop-os/shell#1694] [General]
I work on a big monitor (over 30 inches) and the resizes are too snappy and often I can't get the exact size I want without manually adjusting with my mouse which defeats the purpose of having a tiling window manager. For this reason, I'd like to have the option to adjust the step size at which the windows are resized. Is this possible to do myself by changing a constant, and where is this constant located if it exists?

**Comment by jacobgkau** [comment]
From testing what happens when I change them, I think the constants `hrow` and `hcolumn` in the `resize_auto` function of `src/tiling.ts` are related to what you want to do: https://github.com/pop-os/shell/blob/cf3c932b55d5c247906efe9d81f988ee092d9f51/src/tiling.ts#L498 

However, I see side effects from changing them (e.g. if I halve the numbers from 64 to 32, the resize increment gets cut in half, but I also can't resize either of two windows larger than 50% of the workspace).
